roles/keystone/tasks/main.yml

```diff
@@ -31,6 +31,15 @@
   notify:
     - restart keystone services
 
+- name: install ldap pip reqs in package venv
```

Comment:
We enumerate in the giftwrap manifest all the dependencies we need inside the venv for a particular service. We can express our opinion there, that ldap is required content, even if we don't enable ldap backend in each of our sites. The practice is to consider the venv delivered by giftwrap as immutable, we should not be adding things into the venv unless 100% necessary.